src/components/ChatForm/FilesPreview.tsx

```diff
@@ -11,8 +11,12 @@ export const FilesPreview: React.FC<{
   return (
     <Box p="2" pb="0">
       {files.map((file, i) => {
-        const lineText =
-          file.line1 && file.line2 ? `:${file.line1}-${file.line2}` : "";
+        const lineText = file.cursor
```

Comment:
does the lsp not like it when just one number is given ?

Reply:
LSP even not cares at all about file numbers, we can just send the file path actually and it will work